tyk-pro/templates/statefulset-enterprise-portal.yaml

```diff
@@ -138,6 +138,9 @@ spec:
           - name: enterprise-portal-pvc-{{ include "tyk-pro.fullname" . }}
             mountPath: /opt/portal/themes
             subPath: themes
+          - name: enterprise-portal-pvc-{{ include "tyk-pro.fullname" . }}
+            mountPath: /opt/portal/db
+            subPath: db
```

Comment:
@rewsmith should this by applied only for sqlite?

Reply:
That's a good point @caroltyk.  Let me check and amend as required.

Reply:
Changes made and tested @caroltyk